src/modules/storage/LedgerStorage.ts

```diff
@@ -579,10 +579,11 @@ export class LedgerStorage extends Storages
     }
     /**
      * Get validators
-     * @param height: The height parameter is optional.
+     * @param height The height parameter is optional.
      * validators based on the block height if there is a height.
      * if height null the most up to date state is expected.
      * if the height is null then current valid validators.
+     * @param address The address of the validator
```

Comment:
Likewise:
```suggestion
     * @param address If present, only returns a single validator (or none if not enrolled).
                                     If null, all available validators are returned.
```

If a parameter is optional, you have to describe what are the effect of providing it, or not.